code/modules/worldgen/PrefabRuntimeChecker.dm

```diff
@@ -22,8 +22,7 @@
 /proc/placeAllPrefabs()
 #ifndef RUNTIME_CHECKING
 	CRASH("This proc only works if RUNTIME_CHECKING is defined")
-	return
-#endif
+#else
```

Comment:
won't work.
first needs to be `#if defined(RUNTIME_CHECKING)` for the #else to work